### PR TITLE
Make .dockerize depend on Dockerfile

### PR DESCRIPTION
**Pull request by glyn**
Also .dockerize should be deleted by `make clean`.

### CODE REVIEW

File: function-controller/Makefile [hunk 1]
```diff
@@ -24,11 +24,11 @@ verify-mocks: gen-mocks
 	git diff --exit-code $(GENERATED_SOURCES)
 
 clean:
-	rm -f $(OUTPUT)
+	rm -f $(OUTPUT) .dockerize
 
 dockerize: .dockerize
 
-.dockerize: $(GO_SOURCES) ../vendor
+.dockerize: $(GO_SOURCES) ../vendor ../Dockerfile
```
Comment (ericbottard) [inline function-controller/Makefile]: Should also depend on `../.dockerignore`

**Comment by glyn** [comment]
Fixed with a force push.

Please note I'm leaving debug out of scope for this PR since it's currently pretty broken. :-(

**Comment by ericbottard** [comment]
Why is it broken? I used it no longer than yesterday?

**Comment by glyn** [comment]
Not sure. Let's just say it's out of scope for this PR.

**Comment by ericbottard** [comment]
Merged as ecbdaf9a with additional support for debug-dockerize

**Pull request closed without merging**